docs/content/stable/yugabyte-voyager/release-notes.md

```diff
@@ -17,6 +17,34 @@ What follows are the release notes for the YugabyteDB Voyager v1 release series.
 
 Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`, where `YYYY` is the release year, `M` is the month, and `N` is the number of the release in that month.
 
+## v2025.12.2 - December 30, 2025
+
+### Highlight
+
+- Live migration for PostgreSQL source database (with fall-forward or fall-back using [YugabyteDB Connector](../../additional-features/change-data-capture/using-logical-replication/yugabytedb-connector/)) is {{<tags/feature/ga>}}.
+
+### New features
+
+- The assess-migration command now collects metadata from PostgreSQL primary and replica nodes in parallel, significantly improving assessment performance for multi-node deployments.
+- Added support for User-Defined Types (UDTs) and Array types (including arrays of enums, hstore, and tsvector) with [YugabyteDB Connector](../../additional-features/change-data-capture/using-logical-replication/yugabytedb-connector/) in live migration with fall-back or fall-forward workflows.
+
+### Enhancements
+
+- Enhanced guardrail checks to validate all required permissions in the source database before starting export data. Missing permissions are now clearly reported with specific details, preventing migration failures due to insufficient privileges.
+- Assessment topology in reports. The assessment report now includes topology information about primary and replica nodes assessed, providing better visibility into the scope of the assessment. This information is also sent with call-home diagnostics.
+- Added a new `--primary-only` flag to the assess-migration command for PostgreSQL sources. When specified, the assessment skips read replica discovery and assessment, allowing you to assess only the primary database.
```

Comment:
not added to reference?

Reply:
thanks for pointing this out
cc: @ShivanshGahlot  can you pls provide the reference page details for this flag?